docs/cloud/satellites/byoc/aws/terraform.md

```diff
@@ -0,0 +1,64 @@
+  subnet = "subnet-0123456789abcde01"
+}
+```
+### Module Inputs
+
+| Name           | Description                                                                                                                                              |
+|----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------|
+| cloud_name     | The name to use to identify the cloud installation. Used by Earthly during automatic installation, and to mark related resources in AWS.                 |
+| subnet         | The subnet Earthly will deploy satellites into.                                                                                                          |
+| ssh_public_key | (Optional) The SSH key to include in provisioned satellites. If left unspecified, a new key is generated, and the private key is available as an output. |
+
+
+### Module Outputs
+
+| Name                 | Description                                                                                                                              |
+|----------------------|------------------------------------------------------------------------------------------------------------------------------------------|
+| installation_name    | The name to use to identify the cloud installation. Used by Earthly during automatic installation, and to mark related resources in AWS. |
+| security_group_id    | The ID of the security group for new satellites.                                                                                         |
+| ssh_key_name         | The name of the SSH key in AWS that is included in new satellites.                                                                       |
+| ssh_private_key      | (Sensitive) The private key, if `ssh_public_key` is unspecified.                                                                         |
+| instance_profile_arn | The ARN of the instance profile satellite instances will use for logging.                                                                |
+| compute_role_arn     | The ARN of the role Earthly will assume to orchestrate satellites on your behalf.                                                        |
+
+## Installation
+
+Earthly is able to automatically install BYOC when provisioned via Terraform by running:
+
+```shell
+earthly cloud install --via terraform --name <output-name>
+```
+
+Automatic installation requires some conditions to be met. If both of these conditions cannot be met, you will need to use the [manual](./manual.md) installation method with the infrastructure provisioned by Terraform.
+
+1. The install command is run in the same directory as the module containing your BYOC block.
+2. The `automatic_installation` output is exported under the name specified by `<output-name>`. You could enable automatic installation for the example earlier by adding the following output:
```

Comment:
Is it worth mentioning that `terraform` executable is present, or is that too obvious?

Reply:
I dont know how you got here if you didnt have it